### DISCUSSION TITLE
Database schema, multiple environments

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#7038] [Archive]
Hi all,

Most probably this has been discussed at length but I couldn't really find an answer anywhere.

What is the best way/defacto standard to keep collections schema/database schema etc under source control. And how should I roll out a new environment without data, just collections etc. Should I use the migrations functionality or is there anything else?

I'd like to set up multiple environments (DTAP) with the same schema of course.

Cheers!

**Comment by rijkvanzanten** [comment]
Heya! That sounds like https://github.com/directus/directus/discussions/3891 🙂

**Reply by keesvanbemmel** [reply]
100%, however that didn't really answer what I should do at this moment to migrate stuff the easiest way.

I think I'll try and use/alter the script in this answer: https://github.com/directus/directus/discussions/3891#discussioncomment-906088

Thanks again for the quick reply Rijk!

**Reply by rijkvanzanten** [reply]
There's no way to install a new instance with a starting point schema. The next best thing would be to use a template database dump and import that into your new database during provisioning 👍🏻
